### PR TITLE
Updates Readme to add configration for adding kotest assertions

### PR DESCRIPTION
**Pull request by ashishkujoy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -200,7 +203,9 @@ tasks.withType<Test> {
 }
 
 dependencies {
-  testImplementation("io.kotest:kotest-runner-junit5-jvm:<version>")
+  testImplementation("io.kotest:kotest-runner-junit5-jvm:<version>") // for kotest framework
+  testImplementation("io.kotest:kotest-assertions-core:<version>") // for kotest core assertions
+  testImplementation("io.kotest:kotest-assertions-jvm:<version>") // for kotest jvm assertions
```
Comment (sksamuel) [inline README.md]: We can simplify this to 

 testImplementation("io.kotest:kotest-runner-junit5-jvm:<version>") // for kotest framework
  testImplementation("io.kotest:kotest-assertions-core-jvm:<version>") // for kotest core jvm assertions

File: README.md [hunk 2]
```diff
@@ -183,7 +184,9 @@ android.testOptions {
 }
 
 dependencies {
-    testImplementation 'io.kotest:kotest-runner-junit5:<version>'
+    testImplementation 'io.kotest:kotest-runner-junit5:<version>' // for kotest framework
+    testImplementation 'io.kotest:kotest-assertions-core:<version>' // for kotest core assertions
+    testImplementation 'io.kotest:kotest-assertions-jvm:<version>' // for kotest jvm assertions
```
Comment (sksamuel) [inline README.md]: We can simplify this to

testImplementation("io.kotest:kotest-runner-junit5-jvm:") // for kotest framework
testImplementation("io.kotest:kotest-assertions-core-jvm:") // for kotest core jvm assertions

**Review by sksamuel — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ashishkujoy) [reply README.md]: Done

**Comment by sksamuel** [comment]
:+1:

**Pull request merged**